app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] The association here belongs_to author and not the user then has the class name `User`, it can be similar to `belongs_to :author, class_name: 'User'`. Kindly correct this 🙃 😉

Reply:
Well resolved ✔️